src/Storage/WordPressObjectCacheStorage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Kevinrob\GuzzleCache\Storage;
+
+use Kevinrob\GuzzleCache\CacheEntry;
+
+class WordPressObjectCacheStorage implements CacheStorageInterface
+{
+    /**
+     * @var string
+     */
+    private $group;
+
+    /**
+     * @param string $group
+     */
+    public function __construct($group = 'guzzle')
+    {
+        $this->group = $group;
```

Comment:
WordPress essentially namespaces keys into different groups to help prevent conflicts.  I think a default group of `guzzle` makes sense, but we should allow users to change that if needed.